framework/include/constraints/AutomaticMortarGeneration.h

```diff
@@ -67,7 +67,8 @@ class AutomaticMortarGeneration : public ConsoleStreamInterface
                             const std::pair<BoundaryID, BoundaryID> & boundary_key,
                             const std::pair<SubdomainID, SubdomainID> & subdomain_key,
                             bool on_displaced,
-                            bool periodic);
+                            bool periodic,
+                            bool give_me_wrong_results);
```

Comment:
Yeah, I agree it could be more descriptive, ideally this whole option will be removed soon though. The remaining changes to make correct treatment edge dropping robust really aren't that huge, I just ran out of time. I'll change it.

Reply:
The naming scheme has been changed!